stdlib/public/SwiftShims/LibcShims.h

```diff
@@ -60,10 +60,11 @@ static inline __swift_size_t _swift_stdlib_strlen_unsigned(const unsigned char *
 SWIFT_READONLY
 static inline int _swift_stdlib_memcmp(const void *s1, const void *s2,
                                        __swift_size_t n) {
-#if defined(__APPLE__) || defined(__ANDROID__) || defined(__OpenBSD__)
-  extern int memcmp(const void * _Nullable, const void * _Nullable, __swift_size_t);
+// FIXME: Is there a way to identify Glibc specifically?
+#if defined(__gnu_linux__)
```

Comment:
[I used `__GLIBC__` before](https://github.com/apple/swift-tools-support-core/blob/6f8a6d641d70d6eb563211603b1e74e4752f2bad/Sources/TSCclibc/process.c#L8), doesn't that work?

Reply:
No, because the overlay doesn’t (and cannot, I suppose) include any header from Glibc, so any header-defined macros are not available.